kivymd/uix/filemanager.py

```diff
@@ -469,6 +472,50 @@ def sort_by_name(files):
 
         return sorted_files
 
+    def show_disks(self):
+        if platform == 'win':
+            self.disks = sorted(re.findall(r"[A-Z]+:.*$", os.popen("mountvol /").read(), re.MULTILINE))
+        elif platform == 'linux' or platform == 'android':
```

Comment:
`Linux`, `Mac OS`, `Android` == `Unix`.
So the below code will be correct:

```python
elif platform in ["linux", "android", "macosx"]:
    ...
```

Reply:
Yes)